package/etc/conf.d/conflib/netsource/app-netsource-aruba_clearpass.conf

```diff
@@ -10,10 +10,89 @@ block parser app-netsource-aruba_clearpass() {
             );
         };
 
+        rewrite {            
+            r_set_splunk_dest_update_v2(
+                sourcetype('aruba:clearpass:endpoint:profile')
```

Comment:
@mstopa-splunk I have only 1 concern that wanna discuss.
before this PR we have one generic `sourcetype`=`aruba:clearpss` and on this [issue](https://github.com/splunk/splunk-connect-for-syslog/issues/2300) , customers  add new sourcetypes `aruba:clearpass:something`. 

I worry that will broke behaviour for exsting customers who will upgrade, old logs will be at `aruba:clearpass`, new `aruba:clearpass:something`. **Are we need update only** `class` (`sourcetype` will be `Aruba:clearpass` and `class` will be `something`) or provide for them new sourcetype `Aruba:clearpass:something`

Reply:
After discussion with @mstopa-splunk decided use `sourcetype="aruba:clearpass" class="something"`